components/input/src/auro-input.js

```diff
@@ -144,8 +145,10 @@ export class AuroInput extends BaseInput {
             autocapitalize="${ifDefined(this.autocapitalize ? this.autocapitalize : undefined)}"
             autocomplete="${ifDefined(this.autocomplete ? this.autocomplete : undefined)}"
             part="input"
-            role="${ifDefined(this.role)}"
-            />
+            role="${ifDefined(this.a11yRole)}"
+            aria-expanded="${ifDefined(this.a11yExpanded)}"
+            aria-controls="${ifDefined(this.a11yControls)}"
+          />
```

Comment:
isDefined is not needed as they are defined in properties. 
```suggestion
            role="${this.a11yRole}"
            aria-expanded="${this.a11yExpanded}"
            aria-controls="${this.a11yControls}"
          />
```

Reply:
The use of "ifDefined" acts as a gatekeep to allow us to remove the definition and thereby remove the attribute from the element. They may start as defined, and even be altered later, but we also have a function that resets them to remove the attribute, and removes these attributes manually just to make sure they are gone.